mixer/cmd/bundles.go

```diff
@@ -155,31 +155,19 @@ var bundleListCmd = &cobra.Command{
 
 // Bundle Edit command ('mixer bundle edit')
 type bundleEditCmdFlags struct {
-	copyOnly bool
-	add      bool
-	git      bool
+	add bool
+	git bool
 }
 
 var bundleEditFlags bundleEditCmdFlags
 
 var bundleEditCmd = &cobra.Command{
 	Use:   "edit <bundle> [<bundle>...]",
```

Comment:
create <bundle>  sounds better than edit; the use of edit is now not supported so we should use something more meaningful to what actually happens.

Reply:
@tmarcu Yes, I agree 'edit' does not make sense anymore. 'create' would sound better. I wasn't sure if we should change it right away and maybe could be the next iteration. Also, what are your thoughts on reducing the impact of this PR. E.g. --suppress-editor is removed. Should we have it as deprecated for a while and then remove it completely?